prime-dotnet-webapi/Controllers/LookupsController.cs

```diff
@@ -14,8 +14,7 @@ namespace Prime.Controllers
     [Produces("application/json")]
     [Route("api/[controller]")]
     [ApiController]
-    // User needs at least the READONLY ADMIN or ENROLLEE role to use this controller
-    [Authorize(Policy = Policies.User)]
+    [Authorize(Roles = Roles.PrimeEnrollee + "," + Roles.ViewEnrollee + "," + Roles.ViewSite)]
```

Comment:
I think in this case, using `Roles.PrimeAdministrant` is appropriate, as the lookups are available to every user of PRIME

Reply:
We could also consider just using `[Authorize]` here. No permissions are required, only a valid token. This would mean that our service accounts could access these endpoints, which isn't a bad thing I don't think.

Reply:
I've adjusted to just use `[Authorize]` for `LookupsController`